### PR TITLE
Add links to GitHub source content

### LINKED ISSUE #294: Link to "Edit on GitHub"
Not while this repo is private, or even maybe a blurb on all guide pages, just saying "Edit on GitHub—coming soon".

Just so folks know they can help us with the content.

### PR DESCRIPTION
**Pull request by TimvdLippe**
This is currently based on the author information blocks, which already
have the necessary basic styling. However, we should wait for the CSS to
be deployed before we can iterate on the design.

Fixes #294

### CODE REVIEW

File: templates/glitch.html [hunk 1]
```diff
@@ -45,6 +45,7 @@
         </div>
         <div class="web-codelab-instructions__heading-title">
           <h1>{{title}}</h1>
+          <a href="https://github.com/GoogleChrome/web.dev/tree/master/content{{relatedGuideHref}}/codelab-{{glitch}}.md" target="_blank" rel="noopener">Edit this codelab on GitHub</a>
```
Comment (ebidel) [inline templates/glitch.html]: Nit: maybe use "Edit this page" for everything. More concise and universal across pages.

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by robdodson** [comment]
For the near term I don't think we have a design resource, so we'll have to DIY :P

What if we move it into this region:
screen shot 2019-01-10 at 6 16 11 pm

On smaller screens we may choose to just hide the link, or if you want to get fancy, you can try wrapping it beneath the breadcrumbs.

We may want to remove the blue color and add a pencil icon or something.. Just trying to think of a way to make it fit in with the design of the breadcrumbs.

**Comment by robdodson** [comment]
Just started reviewing :)

**Comment by robdodson** [comment]
Based on discussion with the team, can we do something like this instead?

image

Also you may not want to reuse the breadcrumb class. I noticed on the codelab page it is no longer horizontally centered with the pencil.

Sam suggested changing the text to just "Edit this page".

**Comment by robdodson** [comment]
Actually Hwi suggested we stick with the phrase "Edit this page on GitHub"

**Comment by TimvdLippe** [comment]
@robdodson Do you mind continuing this PR based on the design decision? I am a bit confused as to what we now have landed on and it seemed like you had a version somewhere that already made the magic happen.

File: templates/guide.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <!-- BREADRUMBS AND PROGRESS -->
     <div class="row">
       <div class="breadcrumbs-and-progress">
-        <div class="web-breadcrumbs">
```
Comment (TimvdLippe) [inline templates/guide.html]: Per b/122721396 we should prepend this with `web-`

**Comment by robdodson** [comment]
I'm going to close this pull request. We have designs for this now and I'll add it to the v1 backlog.

**Pull request closed without merging**